conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosDatabaseOperations.scala

```diff
@@ -868,6 +881,62 @@ object TezosDatabaseOperations extends ConseilLogSupport {
         Future.successful(List.empty -> None)
     }
 
+  /** Reads json file for registered tokens */
+  def readJsonFile(network: String): List[RegisteredToken] = {
+    import io.circe.parser.decode
+    import RegisteredTokensFetcher.decoder
+
+    val file = getClass.getResource(s"/registered_tokens/$network.json")
+    val content = Source.fromFile(file.toURI).getLines.mkString
+    decode[List[RegisteredToken]](content) match {
+      case Left(error) =>
+        logger.error(s"Something wrong with registered tokens file $error")
```

Comment:
Lorre should shut down if the content is malformed or we will never realize we don't have proper data.

Reply:
shutting down